io.github._7h0ma5.QLog.metainfo.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2020 hamradio lu9dce -->
+<component type="desktop-application">
+ <id>io.github._7h0ma5.QLog</id>
+ <metadata_license>CC0-1.0</metadata_license>
+ <project_license>GPL-3.0</project_license>
+ <name>Qlog</name>
+ <summary>QLog is an Amateur Radio logging</summary>
+ <developer_name>Thomas Gatzweiler</developer_name>
+ <url type="homepage">https://github.com/7h0ma5/QLog</url>
+ <description>
+  <p>
+  QLog is an Amateur Radio logging application for Linux, Windows and Mac OS.
+  It is based on the Qt 5 framework und uses SQLite as database backend.
+  </p>
+ </description>
+ <screenshots>
+  <screenshot type="default">
+   <image>https://camo.githubusercontent.com/2cad7fe5bdd8addba5c023f2b095f751a4b2c4e1d902a97000ab6d625e3d4025/68747470733a2f2f716c6f672e646c3269632e64652f73637265656e73686f742e706e67</image>
+  </screenshot>
+ </screenshots>
+ <releases>
+   <release version="1.0" date="2020-11-1"/>
```

Comment:
Just a minor issue that I noticed: The date format should probably be 2020-11-**0**1.

Reply:
Yes, this needs fixing. `1.0` is not correct either, it should be something along `0.0.<date of the commit>`.